Projects/hCoreUI/Sources/Styling/DefaultStyling.swift

```diff
@@ -198,10 +198,10 @@ extension DefaultStyling {
             for: .highlighted
         )
 
-        UIDatePicker.appearance().tintColor = .brand(.link)
+        UIDatePicker.appearance().tintColor = .brandNew(.primaryText())
```

Comment:
@sladan-hedvig not sure if we should just change the date picker color for both old and new design like this or somehow check if new design is enabled?

Reply:
Returned to previous value. Updated tint color in the place

Reply:
@sladan-hedvig but date picker is still black for old design. should it be that?